clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/agent/KubernetesCoreCachingAgent.java

```diff
@@ -60,4 +73,9 @@ public KubernetesCoreCachingAgent(
   protected List<KubernetesKind> primaryKinds() {
     return credentials.getGlobalKinds();
   }
+
+  @Override
+  protected boolean cachesKind(KubernetesKind kind) {
+    return kind.getApiGroup().isNativeGroup();
```

Comment:
Similar request for comments as https://github.com/spinnaker/clouddriver/pull/5638/files#r812228254.